docs/documentation-guide.md

```diff
@@ -0,0 +1,18 @@
+
+
+## Where can I find documentation?
+
+The application has built-in documentation web pages, you will see them when you opened the application web pages or you can simply open `html/index.html` or just use http://api.asksusi.com as reference. 
+
+## Where can I find showcases and tutorials?
+
+Articles and tutorials are also on our blog at http://blog.loklak.net.
```

Comment:
@Abhi2424shek I m solving issue #6 also.So i think i need to change it or shall i submit a new PR for that after this.